### ISSUE TITLE
Mutating images in weird ways

### ISSUE THREAD

**Issue by krainboltgreene** [amercier/broccoli-file-size#1]
Apparently running things through broccoli-file-size causes image files to be broken?

I have a repository where I can reproduce the situation.

**Comment by amercier** [comment]
@krainboltgreene Interesting, thanks for reporting this! I'll try to investigate this when I have bandwidth.

**Comment by amercier** [comment]
I expect #6 to fix this once implemented.

**Comment by krainboltgreene** [comment]
Er...Is that a very good idea? A symlink can't be deployed to S3.

**Comment by amercier** [comment]
@krainboltgreene Many Broccoli plugins already make use of symlinking (see [broccoli-funnel](https://github.com/broccolijs/broccoli-funnel/blob/master/index.js#L388) for instance). So I believe it should try symlinking, but fallback to copying when not available.

**Comment by krainboltgreene** [comment]
Fair enough!

**Comment by krainboltgreene** [comment]
I appreciate you working on this, btw. <3

**Comment by amercier** [comment]
Thanks! I appreciate your feedbacks too :+1:

**Comment by amercier** [comment]
Should be fixed in v1.1.0:

```
> broccoli-file-size@1.1.0 test [...]/broccoli-file-size
> rm -rf temp && broccoli build temp && mocha

broccoli-logo.generated.png => 7.46 kB (7.15 kB gzipped)
lorem.txt => 446 B (284 B gzipped)


  ✓ does not alter text files
  ✓ does not alter binary files

  2 passing (11ms)
```

Feel free to re-open this ticket if this is not the case.
